packages/api-gateway/__tests__/places/getPlaces.test.js

```diff
@@ -0,0 +1,16 @@
+const { fastify } = require('../../src/drivers/http/server');
+
+// At least in my pc it lasts almost 15s
+jest.setTimeout(120000);
+
+describe('Given a client that wants to get the Places data', () => {
+  describe('When it does the GET petition', () => {
+    test('Then the back must read it from the db and return it', async () => {
+      const response = await fastify.inject({
+        method: 'GET',
+        url: 'places',
+      });
+      expect(response.statusCode).toBe(200);
+    });
```

Comment:
@Alejandrocuartas seems that we are just testing happy paths, what happens if the place does not exist?

Reply:
That route is for getting the places, the route for getting a specific place is other [issue](https://github.com/Platzi-Master-C9/booking-services/issues/34).

Reply:
yes indeed, but we need to bear in mind in tests also cases where things are not expected to work.